.github/workflows/release-please.yml

```
@@ -0,0 +1,36 @@
+name: Release
+on:
+  push:
+    branches:
+      - 'ci/migrate-to-release-please'
+
+jobs:
+  release:
+    name: Create release
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@ee9822ec2c397e8a364d634464339ac43a06e042 # v3
+        id: release
+        with:
+          command: manifest
+          default-branch: "ci/migrate-to-release-please"
+
+  publish:
```

NITS: Please specify the job name as same as the release job.

```suggestion
  publish:
    name: Publish to npm
```